### PR TITLE
Add links to RSS feed and clean up feed

### PR DESCRIPTION
**Pull request by frodelu**
Added links for RSS feed auto-discovery. Also changed feed description to use excerpt instead of entire content.

@mpolden: Please review. FYI: @kkraune .

### CODE REVIEW

**Review by kkraune — APPROVED** [review]
great!

File: feed.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@ layout: null
     {% for post in site.posts limit:10 %}
       <item>
         <title>{{ post.title | xml_escape }}</title>
-        <description>{{ post.content | xml_escape }}</description>
+        <description>{{ post.excerpt | strip_html | xml_escape }}</description>
```
Comment (mpolden) [inline feed.xml:17]: Can we keep including the entire post plz?

Comment (frodelu) [reply feed.xml:17]: PTAL now.

**Review by kkraune — APPROVED** [review]

**Review by mpolden — COMMENTED** [review]
Great!

**Pull request merged**